packages/web-components/src/components/va-text-input/va-text-input.tsx

```diff
@@ -140,6 +140,11 @@ export class VaTextInput {
    */
   @Prop() success?: boolean = false;
 
+  /**
+   * Displays the input at a specific width. Accepts 2xs (4ex), xs (7ex), sm or small (10ex), md or medium (20ex), lg (30ex), xl (40ex), and 2xl (50ex).
+   */
```

Comment:
This description was taken directly from USWDS: https://designsystem.digital.gov/components/text-input/#component-variants-text-input